src/lib/containers/table/SynapseTable.tsx

```diff
@@ -158,17 +169,30 @@ export default class SynapseTable extends React.Component<
       mapUserIdToHeader: {},
       isFetchingEntityHeaders: false,
       isFetchingEntityVersion: false,
+      tableElementId: uniqueId()
     }
     this.getEntityHeadersInData = this.getEntityHeadersInData.bind(this)
   }
+  
+  // instance variables
+  resizer:any
```

Comment:
there are times when we have two tables on the same page, does have the variable declared like this pose any problem? I think it would be fine right?

Reply:
should be ok (if the variable really is associated to the instance!).